kubeflow/trainer/rhai/traininghub.py

```
@@ -369,12 +411,15 @@ def _transform_osft(self, metrics):
             step_total = steps_per_epoch * total_epochs if steps_per_epoch > 0 else 0
 
             current_step_absolute = step
-            percent = min(100, (current_step_absolute / step_total * 100)) if step_total > 0 else 0
+            progress = (current_step_absolute / step_total * 100) if step_total > 0 else 0
+            percent_int = int(round(progress))
```

Ruff (0.15.7)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)